tests/SprykerSdkTest/Spryk/Console/SprykRunSpryksTest.php

```diff
@@ -0,0 +1,95 @@
+<?php
+
+/**
+ * Copyright © 2016-present Spryker Systems GmbH. All rights reserved.
+ * Use of this software requires acceptance of the Evaluation License Agreement. See LICENSE file.
+ */
+
+namespace SprykerSdkTest\Spryk\Console;
+
+use Codeception\Test\Unit;
+use SprykerSdk\Spryk\Console\SprykRunConsole;
+
+/**
+ * Auto-generated group annotations
+ *
+ * @group SprykerSdkTest
+ * @group Spryk
+ * @group Console
+ * @group SprykRunSpryksTest
+ * Add your own group annotations below this line
+ */
+class SprykRunSpryksTest extends Unit
+{
+    /**
+     * @var \SprykerSdkTest\SprykConsoleTester
+     */
+    protected $tester;
```

Comment:
```suggestion
    protected SprykConsoleTester $tester;
```
Not a blocker but we should use it where possible.